psamm_import/importer.py

```diff
@@ -556,6 +562,12 @@ def main():
         logging.basicConfig(
             level=logging.INFO, format=u'%(levelname)s: %(message)s')
 
+    if(os.path.isfile(args.dest + '/model.yaml')and not args.force):
+        raise OverwriteError('Files will be overwritten, but if you want to'
+                             ' continue with the import add `--force` to your'
+                             ' command.')
+        return
+
```

Comment:
This only checks if `model.yaml` exists but other output files could still be overwritten. I think the best approach would be requiring `--force` if the directory is not empty. I think you can use `os.listdir()` to check if it has files. I would report the error as something like: "Destination directory is not empty. Use --force option to proceed anyway, overwriting any existing files in {}".format(dest). Report the error with `logger.error()` followed by return 1. This will become the error code from the program.